orttraining/orttraining/python/training/ortmodule/__init__.py

```diff
@@ -0,0 +1,25 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+from packaging import version
+
+# All global constant goes here, before ORTModule is imported
+ONNX_OPSET_VERSION = 12
+MINIMUM_TORCH_VERSION_STR = '1.8.1'
+
+from .ortmodule import ORTModule
+
+
+# Verify proper PyTorch is installed before proceding to ONNX Runtime initializetion
+try:
+    import torch
+    torch_version = version.parse(torch.__version__.split('+')[0])
+    minimum_torch_version = version.parse(MINIMUM_TORCH_VERSION_STR)
+    if torch_version < minimum_torch_version:
+        raise RuntimeError(
+            f'ONNXRuntime ORTModule frontend requires PyTorch version greater or equal to {MINIMUM_TORCH_VERSION_STR}, '
+            f'but version {torch.__version__} was found instead!')
```

Comment:
nit: Can we avoid the exclamation at the end, sounds a bit alarming :P

Reply:
That is an exception crashing user's training, shouldn't it be alarming? lol

If more changes are required, I can bundle them together to save CI. Otherwise, I can do on my next PR which is blocked by this

Reply:
I have updated it :)